dev/com.ibm.ws.http2.client/src/com/ibm/ws/http2/test/Http2Client.java

```diff
@@ -370,16 +399,27 @@ private long sendFrame(Frame writableFrame, long timeout, boolean bypassPreface)
 
     public Http2Client waitFor(Frame expectedFrame) {
         //check once if the same already arrived... then we will use the listener to avoid doing the search.
+        // This is not Thread safe
```

Comment:
Any concern here?

Reply:
Don't think we have anything to worry about here, I believe I added this for future proofing to let the dev know that this isn't thread safe if I remember correctly